data/tabular/blood_brain_barrier_martins_et_al/transform.py

```diff
@@ -138,61 +138,56 @@ def get_and_transform_data():
+Assistant: {penetrate_BBB#No&Yes}, this molecule can {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.""",  # noqa: E501
+            """User: Is the molecule with the {SMILES__description} {SMILES#} able to {penetrate_BBB__names__adjective}?
+Assistant: {penetrate_BBB#No&Yes}, it is {penetrate_BBB#not &NULL}able to {penetrate_BBB__names__adjective}.""",  # noqa: E501
+            """User: Can you give me the {SMILES__description} of a molecule that is {penetrate_BBB#not &NULL}able to {penetrate_BBB__names__adjective}?
 Assistant: Yes, here you go: {SMILES#}""",  # noqa: E501
-            """User: I'm looking for the {SMILES__description} of a molecule that is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}?
-Assistant: This is a molecule that is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}: {SMILES#}""",  # noqa: E501
+            """User: I'm looking for the {SMILES__description} of a molecule that can {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}?
+Assistant: This is a molecule that can {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}: {SMILES#}""",  # noqa: E501
             """User: I want to create a molecule {SMILES__description}.
 Assistant: This sounds very exciting! Should I consider any constraints during generation?
 User: Yes, please. The molecule should {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.
-Assistant: Ok, this {SMILES__description} is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}: {SMILES#}""",  # noqa: E501
+Assistant: Ok, this {SMILES__description} can {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}: {SMILES#}""",  # noqa: E501
             """User: I want to create a molecule {SMILES__description}.
 Assistant: This sounds very exciting! Should it be a special molecule?
-User: Yes, the molecule should {penetrate_BBB#not &NULL}be {penetrate_BBB__names__adjective}.
-Assistant: Got it, this {SMILES__description} is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}: {SMILES#}""",  # noqa: E501
+User: Yes, the molecule should {penetrate_BBB#not &NULL}be able to {penetrate_BBB__names__adjective}.
+Assistant: Got it, this {SMILES__description} is {penetrate_BBB#not &NULL}able to {penetrate_BBB__names__adjective}: {SMILES#}""",  # noqa: E501
             # Benchmarking text templates
-            "Is the {SMILES__description} {SMILES#} {penetrate_BBB__names__adjective}:<EOI> {penetrate_BBB#yes&no}",  # noqa: E501 for the benchmarking setup <EOI> separates input and output
+            "Does the {SMILES__description} {SMILES#} {penetrate_BBB__names__adjective}:<EOI> {penetrate_BBB#yes&no}",  # noqa: E501 for the benchmarking setup <EOI> separates input and output
             # todo: check if we go for multiple choice only and remove the benchmarking template above and below
-            """Task: Please classify a molecule based on the description.
```

Comment:
This was the same text template but with benchmark separator `<EOI>`, but maybe I'm missing something?